### ISSUE TITLE
The result is much worse than the paper

### ISSUE THREAD

**Issue by chen-yingfa** [ruinunca/PGTask#1]
I am trying to finetune a distilgpt2 using the following command, but the result is much worse than what was given in the paper. Is there anything I'm missing?

```bash
python run_profile_generation.py --do_train --do_eval --model_name_or_path distilgpt2
```

Thanks in advance,
Donny Chan

**Comment by chen-yingfa** [comment]
Update: I have also tried setting different learning rates and batch sizes, but it's still much worse than the result in the paper.

**Comment by ruinunca** [comment]
Hi Donny,

Have you followed the steps from the Evaluation section (generate_results + evaluate_results)?
Is the loss decreasing?

Rui Ribeiro

**Comment by chen-yingfa** [comment]
Hi Rui!

I think I have found the reason, it's because the latest commit have commented out the correct `create_input` function that includes a `GEN_TOKEN` in `generate_results.py` (probably a result of temporary testing), and the current `create_input` does not append a `GEN_TOKEN` to the input sentence. The result is that the model have a strong tendency to generate "." (a period).

Thanks for responding
Donny Chan

**Comment by ruinunca** [comment]
Thank you, I have fixed the issue on the latest version.
